supplementary/alica_ros1/alica_tests/libalica-tests/include/libalica-tests/util/AlicaTestsUtilityFunction.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <engine/BasicUtilityFunction.h>
```

Comment:
This isn't necessary, you can directly link to BasicUtilityFunction as is now done in lbc/my turtlesim PR

Reply:
In LBC these helper classes are still used as far as I know. All UtilityFunctions with a custom getUtilityFunction() implementation need this helper class, otherwise it wont make use of the getUtilityFunction() implementation. The helper class prevents the creation of separate create functions for each UtilityFunction.